### ISSUE TITLE
Review Flask App 

### ISSUE THREAD

**Issue by St-d603** [allyharrison/Agile-Web-Dev-CITS5505#29]
If the upcoming flask is merged could someone look at sub-dividing the app.py per lecture tutorial (19th), if possible or ask a tutorial, I think it may look more efficient/ in-line with practices if it is divided up? I'm not 100% sure on this so let me know if you have any thoughts.

**Comment by allyharrison** [comment]
Yeah I agree -  I can make it the next thing that I work on, if that's helpful?
I can get onto it once I wrap up the frontend stuff that I've been doing for the first deliverable today/tomorrow.

Rearranging the Flask stuff might help me get my thoughts around it a bit more - it would tie in nicely with the forms I've been styling, plus I was hoping to move a couple of things around with the Jinja blocks  and HTML anyway.
